src/api/scatter.js

```diff
@@ -1,22 +1,45 @@
 import ScatterJS from 'scatterjs-core';
-import ScatterEOS from 'scatterjs-plugin-eosjs';
-import Eos from 'eosjs';
+import ScatterEOS from 'scatterjs-plugin-eosjs2';
+import {JsonRpc, Api} from 'eosjs';
+import EosApi from 'eosjs-api';
 import * as config from '@/config';
 import PriceFormatter from './priceFormatter';
 
-ScatterJS.plugins(new ScatterEOS());
-
-// const account = 'signature.bp';
+// for eosjs-api
+const options = {
+  httpEndpoint: 'https://api.eosnewyork.io:80', // great one
```

Comment:
HTTPS 应该走的 `:443`，我们删掉 `:80`，让浏览器自己连接port 443
```suggestion
  httpEndpoint: 'https://api.eosnewyork.io', // great one
```

Reply:
好的